dist/jquery.flot.js

```diff
@@ -807,7 +807,9 @@ Licensed under the MIT license.
                     tickDecimals: null, // no. of decimals, null means auto
                     tickSize: null, // number or [number, "unit"]
                     minTickSize: null, // number or [number, "unit"]
-                    offset: { below: 0, above: 0 } // the plot drawing offset. this is calculated by the flot.navigate for each axis
+                    offset: { below: 0, above: 0 }, // the plot drawing offset. this is calculated by the flot.navigate for each axis
+                    centerLeft: null,
```

Comment:
why not 0 ?

0 will make all the "axis.centerLeft ? axis.centerLeft : 0" in the code unnecessary